makefiles/aircrack-ng.mk

```
@@ -0,0 +1,52 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += aircrack-ng
+AIRCRACK-NG_COMMIT  := 482f56c5a1433df6b8619f1de6d89d2d5f5ab743
+AIRCRACK-NG_VERSION := 1.6~20211007.$(shell echo $(AIRCRACK_COMMIT) | cut -c -7)
+DEB_AIRCRACK-NG_V   := $(AIRCRACK-NG_VERSION)
+
+aircrack-ng-setup: setup
+	$(call GITHUB_ARCHIVE,aircrack-ng,aircrack-ng,$(AIRCRACK-NG_COMMIT),$(AIRCRACK-NG_COMMIT))
+	$(call EXTRACT_TAR,aircrack-ng-$(AIRCRACK-NG_COMMIT).tar.gz,aircrack-ng-$(AIRCRACK-NG_COMMIT),aircrack-ng)
+	$(call DO_PATCH,aircrack-ng,aircrack-ng,-p1)
+	rm $(BUILD_WORK)/aircrack-ng/lib/libac/support/strlcat.c && touch $(BUILD_WORK)/aircrack-ng/lib/libac/support/strlcat.c
+	rm $(BUILD_WORK)/aircrack-ng/lib/libac/support/strlcpy.c && touch $(BUILD_WORK)/aircrack-ng/lib/libac/support/strlcpy.c
```

This can be done in one line.
```suggestion
	rm $(BUILD_WORK)/aircrack-ng/lib/libac/support/{strlcat.c,strlcpy.c} && touch $(BUILD_WORK)/aircrack-ng/lib/libac/support/{strlcat.c,strlcpy.c}
```